### PR TITLE
Fat 793 tc c455

### PR DESCRIPTION
**Pull request by dmitriyfolio**

### CODE REVIEW

File: cypress/integration/fee-fine/newFeeFines.spec.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            testData.servicePointId = res[0].id;
+            UserEdit.addServicePointViaApi(testData.servicePointId, testData.userProperties.id);
+            UsersOwners.createViaApi({ owner: uuid() }).then(owner => {
+              testData.owner = { id : owner.id, name : owner.ownerName };
+              ManualCharges.createViaApi({ ...ManualCharges.defaultFeeFineType, ownerId: owner.id }).then(manualCharge => {
+                testData.feeFineType = { id:  manualCharge.id, feeFineTypeName: manualCharge.feeFineType };
+                PaymentMethods.createViaApi(testData.owner.id).then(createdPaymentMethod => {
+                  testData.paymentMethod = { id: createdPaymentMethod.id, name: createdPaymentMethod.name };
+                  // TODO: clarify why initial login is needed to load accordion with Fee/Fines
+                  // cy.loginAsAdmin({ path: AppPaths.getUserPreviewPath(testData.userId), waiter: UsersCard.waitLoading });
```
Comment (MaksimTerekhov) [inline cypress/integration/fee-fine/newFeeFines.spec.js:42]: Do we need this comment?

File: cypress/support/fragments/users/newFeeFine.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  close: () => cy.do(rootModal.find(Button({ id :'cancelCharge' })).click()),
+  setFeeFineType: (feeFineType) => {
+    // TODO: fix iterators issue related with select
+    cy.get('div[id=new-modal] select[name=feeFineId]').select(feeFineType);
+  },
+  checkAmount: (amount) => {
+    cy.expect(amountTextField.has({ value : amount.toFixed(2) }));
+    cy.expect(amountTextField.has({ disabled: false }));
+  },
+  cancel:() => cy.do(rootModal.find(Button({ id:'cancelCharge' })).click()),
```
Comment (IhorBohdan) [inline cypress/support/fragments/users/newFeeFine.js:37]: Move to const button

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/users/payFeeFaine.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+export default {
+  waitLoading:() => {
+    cy.expect(rootModal.exists());
+  },
+  checkAmount:(amount) => cy.expect(amountTextfield.has({ value: amount.toFixed(2) })),
+  setPaymentMethod: ({ name: paymentMethodName }) => cy.get('div[class^=modal-] select[name=method]').select(paymentMethodName),
+  submit: () => cy.do(rootModal.find(Button({ id:'submit-button' })).click()),
+  confirm:() => cy.do(Modal('Confirm fee/fine payment').find(Button({ id: matching('clickable-confirmation-[0-9]+-confirm') })).click()),
+  checkPartialPayConfirmation:() => cy.expect(Modal('Confirm fee/fine payment').find(HTML(including('will be partially paid'))).exists),
```
Comment (IhorBohdan) [inline cypress/support/fragments/users/payFeeFaine.js:14]: Move modal to const

File: cypress/support/fragments/users/usersCard.js [hunk 4]
```diff
@@ -19,6 +20,9 @@ export default {
   openLoans() {
     cy.do(Accordion({ id : 'loansSection' }).clickHeader());
   },
+  openFeeFines() {
+    cy.do(Accordion({ id : 'accountsSection' }).clickHeader());
```
Comment (IhorBohdan) [inline cypress/support/fragments/users/usersCard.js:24]: move to const

File: cypress/integration/fee-fine/newFeeFines.spec.js (hunk 1 above)
Comment (dmitriyfolio) [reply cypress/integration/fee-fine/newFeeFines.spec.js:42]: yes, there are few questions/problems which should be clarified in the future

File: cypress/support/fragments/users/newFeeFine.js (hunk 2 above)
Comment (dmitriyfolio) [reply cypress/support/fragments/users/newFeeFine.js:36]: currently it used in one method only, no reasons to create special constant now

**Pull request merged**